buku.py

```diff
@@ -2946,6 +2964,21 @@ def prep_tag_search(tags):
 
     return tags, search_operator, excluded_tags
 
+def wrap_in_case_statement(fragment):
+    """Wraps SQL fragment in CASE statements.
+
+    Parameters
+    ----------
+    fragment : str
+        An SQL fragment
+
+    Returns
+    -------
+    str
+        SQL case statement containing the fragment.
+    """
+
+    return "CASE WHEN " + fragment + " THEN 1 ELSE 0 END"
```

Comment:
Does it need to be a function? It seems way too big for single line of code

Maybe use lambda?

Reply:
👍 makes sense. Updated accordingly.